aip/general/0180.md

```diff
@@ -139,6 +139,84 @@ code, as such changes will be seen as breaking by those users.
 whether a proposed change is likely to break users, and an expansive reading of
 this guidance could ostensibly prevent _any_ change (which is not the intent).
 
+#### Default values must not change
+
+Changing the default value is considered breaking and **must not** be done. The
```

Comment:
I think we need an introductory paragraph, exaplaining:

- What's meant by "default value"
- The scope of the section

In terms of scope, consider list operations - if the user doesn't specify a page size, the server can pick one. That could be regarded as a default value, but it seems reasonable for *that* to change. I suspect this section should basically be scoped to (and explained as) "the default values set by the server for fields in resources, when they're not specified by the client".

Reply:
Adding a section here specifying defaults values within resources, and that necessarily "dynamic defaults" aren't subject to this restriction. Values like a default IP address obviously have to change, this only applies to "static" defaults like instance type or a default integer value